### ISSUE TITLE
Add "npm run lint:fix" auto before "git add ."

### ISSUE THREAD

**Issue by Ruiub3i** [bepronetwork/bepro-js#60]
**Describe the bug**
Add "npm run lint:fix" auto before "git add ."

**Reproduction**

1. Run "git add ."
2. Automatically run "npm run lint:fix" before

**Logs, screenshots**

Goal is to remove this issue

<img width="663" alt="Screenshot 2021-05-21 at 13 46 51" src="https://user-images.githubusercontent.com/26004658/119139496-fff44480-ba3a-11eb-9c25-f5b07bb0d26a.png">

**Comment by nodexchange** [comment]
Hey, I am a little stuck with this one, as they are no git hooks I can tap into to get it sorted. 
https://git-scm.com/docs/githooks

I had a play, I've added smarter lint-staged and husky for us to make it more scalable in the future, but as it stands we might not be able to resolve it. PR on the way. Also, I've added a troubleshoot.md, which we should all contribute too, I am sure it will save you a lot of time :D
